include/RAJA/RAJA.hxx

```diff
@@ -65,39 +65,35 @@
 
 #include "RAJA/internal/defines.hxx"
 
-#include "RAJA/int_datatypes.hxx"
-#include "RAJA/real_datatypes.hxx"
+#include "RAJA/internal/int_datatypes.hxx"
```

Comment:
Should we make a RAJA/Types.hxx file and pull the int and real type stuff into it?  I would think users would want to see this, so it shouldn't be in "internal".